### ISSUE TITLE
So, is the "virtual terminals" enabled in gtel3g kernel???

### ISSUE THREAD

**Issue by MateusL13** [realaravinth/pmOS-samsung-gtel3g#1]
I'm using pmos on my gtel3g, but xfce only starts over ssh (like is said on the gtel3g page) with "sudo startx" or "sudo startxfce4".
While I was searching for a fix, I found that: "Enable virtual terminals in your kernel config." https://wiki.postmarketos.org/wiki/Display_manager
I think this can be the problem, but I don't know how to check...
Also, is there any hardware acceleration working on gtel3g???

**Comment by realaravinth** [comment]
Hello :wave: 

Not sure about virtual terminals, but hardware acceleration isn't working on gtel3g

**Comment by MateusL13** [comment]
> Hello 👋
> 
> Not sure about virtual terminals, but hardware acceleration isn't working on gtel3g

So, I fixed xfce, it now starts automatically without ssh or "sudo startx"!!!

Do you know any way of getting hardware acceleration??

**Comment by realaravinth** [comment]
> Do you know any way of getting hardware acceleration??

No, sorry. I recommend asking on the main chatroom, the people there are very helpful
